### PR TITLE
Add _.getWith function

### PR DESCRIPTION
**Pull request by sharmaaditya570191**
Takes customizer to customize values retrieved from _.get

### CODE REVIEW

**Comment by sharmaaditya570191** [comment]
Hi @jdalton . How should I resolve the conflicts?

File: lodash.js [hunk 1]
```diff
@@ -13128,6 +13128,34 @@
       return result === undefined ? defaultValue : result;
     }
 
+    /**
+     * This method is similat to '_.get' except that it accepts `customizer`
```
Comment (georgeglessner) [inline lodash.js]: similat -> similar

**Comment by sharmaaditya570191** [comment]
Hi @georgeglessner .Thanks for the review. I have corrected the typo.

**Comment by jdalton** [comment]
Hi @sharmaaditya570191!

In this case the customizer is just mapping the return value which isn't really what our `with` methods do. In this case I think this fits better as a one-off helper module in your project/npm.

**Comment by sharmaaditya570191** [comment]
Hi @jdalton . Where do I need to move this in order to add this in project/npm.

**Pull request closed without merging**